lib/top_hangman/word.rb

```diff
@@ -0,0 +1,46 @@
+module TopHangman
+  class Word
+    WORDS_FILE_SOURCE = 'lib/google-10000-english-no-swears.txt'.freeze
+    MIN_WORD_SIZE = 5
+    MAX_WORD_SIZE = 12
+
+    attr_reader :current_word
+
+    def initialize
+      read_words_file(WORDS_FILE_SOURCE)
+      @history_of_words = []
+      @current_word = nil
+    end
+
+    def set_current_word
+      @current_word = random_word
+      @history_of_words << @current_word
+    end
```

Comment:
delete this. Setting the current word is responsibility of the round. At some point it will have to call `@current_word = Word.from_file`